lua/entities/gmod_wire_cpu.lua

```diff
@@ -4,8 +4,18 @@ ENT.PrintName       = "Wire ZCPU"
 ENT.Author          = "Black Phoenix"
 ENT.WireDebugName	= "ZCPU"
 
+CreateConVar("wire_cpu_max_frequency", "1400000", FCVAR_REPLICATED)
```

Comment:
I think this ConVar doesn't belong on the clientside. So, move it below (on the line 12).
And then use the `cpu_max_frequency` value as a default ConVar value (that is, the second argument can be a number/string).